x-pack/plugins/security_solution/public/entity_analytics/translations.ts

```diff
@@ -170,14 +170,15 @@ export const UPDATE_RISK_ENGINE_MODAL_EXISTING_USER_HOST_2 = i18n.translate(
 export const UPDATE_RISK_ENGINE_MODAL_EXISTING_DATA_1 = i18n.translate(
   'xpack.securitySolution.riskScore.updateRiskEngineModal.existingData_1',
   {
-    defaultMessage: 'None of your risk score data will be deleted',
+    defaultMessage: 'Legacy risk score data will not be deleted',
```

Comment:
Should we generalize this? "Legacy" might not mean anything to the user here: 
```suggestion
    defaultMessage: 'Existing risk score data will not be deleted',
```